Mathlib/RingTheory/Valuation/PrimeMultiplicity.lean

```diff
@@ -1,23 +1,97 @@
 /-
 Copyright (c) 2018 Robert Y. Lewis. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Robert Y. Lewis, Chris Hughes
+Authors: Robert Y. Lewis, Chris Hughes, Daniel Weber
 -/
 import Mathlib.RingTheory.Multiplicity
+import Mathlib.RingTheory.Localization.NumDen
 import Mathlib.RingTheory.Valuation.Basic
+import Mathlib.RingTheory.Valuation.ExtendToLocalization
+import Mathlib.RingTheory.UniqueFactorizationDomain.Multiplicity
 
 /-!
 # `multiplicity` of a prime in an integral domain as an additive valuation
 -/
 
 variable {R : Type*} [CommRing R] [IsDomain R] {p : R}
 
+namespace AddValuation
+
 /-- `multiplicity` of a prime in an integral domain as an additive valuation to `ℕ∞`. -/
-noncomputable def multiplicity_addValuation (hp : Prime p) : AddValuation R ℕ∞ :=
+noncomputable def multiplicity (hp : Prime p) : AddValuation R ℕ∞ :=
```

Comment:
Should this maybe be made `protected` since I can imagine we'd want to talk about the original `multiplicity` definition too?
```suggestion
noncomputable protected def multiplicity (hp : Prime p) : AddValuation R ℕ∞ :=
```

Reply:
Good idea, done